resources/js/components/data-list/HasFilters.js

```diff
@@ -25,25 +25,40 @@ export default {
     },
 
     created() {
-        this.$events.$on('filters-changed', this.filtersChanged);
+        this.$events.$on('filter-changed', this.filterChanged);
         this.$events.$on('filters-reset', this.filtersReset);
     },
 
     methods: {
 
+        hasFields(values) {
+            for (const fieldHandle in values) {
+                if (values[fieldHandle]) return true;
```

Comment:
This is the only part that was a bit iffy to me.

If, for example, your filter wants to have a null or false value, it will be treated as missing.

I think that's probably fine. If it becomes an issue down the road we can tackle it then.